### PR TITLE
added frame count to FrameTimeDiagnosticsPlugin

### PR DESCRIPTION
**Pull request by Olaren15**
see issue #193

### CODE REVIEW

**Review by joshuajbouw — APPROVED** [review]
Looks good and simple enough to include to me. Good improvement.

File: crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs [hunk 1]
```diff
@@ -3,32 +3,47 @@ use bevy_app::prelude::*;
...
         app.add_startup_system(Self::setup_system.system())
+            .add_resource(FrameTimeDiagnosticsState { frame_count: 0.0 })
             .add_system(Self::diagnostic_system.system());
     }
 }
 
 impl FrameTimeDiagnosticsPlugin {
     pub const FPS: DiagnosticId = DiagnosticId::from_u128(288146834822086093791974408528866909483);
+    pub const FRAME_COUNT: DiagnosticId =
+        DiagnosticId::from_u128(54021991829115352065418785002088010277);
```
Comment (cart) [inline crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs:25]: i think id prefer a fully-random uuid just so these are more distinguishable when printed. I generated one for you :)

`73441630925388532774622109383099159699`

File: crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs [hunk 2]
```diff
@@ -3,32 +3,47 @@ use bevy_app::prelude::*;
...
+        mut diagnostics: ResMut<Diagnostics>,
+        time: Res<Time>,
+        mut state: ResMut<FrameTimeDiagnosticsState>,
+    ) {
         if time.delta_seconds_f64 == 0.0 {
             return;
         }
 
+        state.frame_count += 1.0;
+        diagnostics.add_measurement(Self::FRAME_COUNT, state.frame_count);
```
Comment (cart) [inline crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs:45]: i think we should probably move this above the `time.delta_seconds_f64 == 0.0` check. A frame still happens even if the delta time is 0.

**Review by cart — APPROVED** [review]

**Comment by mockersf** [comment]
> why use an f64 instead of a u64

All diagnostics are saved as `f64` for [`add_measurement`](https://docs.rs/bevy/0.5.0/bevy/diagnostic/struct.Diagnostics.html#method.add_measurement)

**Comment by billyb2** [comment]
@mockersf oh okay, thank you

**Pull request merged**